backend/src/main/java/gov/cdc/usds/simplereport/config/OktaLocalCorsSecurityConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package gov.cdc.usds.simplereport.config;
+
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.servlet.config.annotation.CorsRegistration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@Profile("!" + BeanProfiles.NO_SECURITY + " & " + BeanProfiles.CREATE_SAMPLE_DEVICES)
+@ConditionalOnWebApplication
+@Slf4j
+public class OktaLocalCorsSecurityConfiguration implements WebMvcConfigurer {
```

Comment:
It seems like the content of this one is the same as `CorsSecurityConfiguration`, could we just use that one for Okta local as well by changing it to just `@Profile("!" + BeanProfiles.NO_SECURITY)`?

Reply:
that's a good point, I wanted to keep the settings 1 to 1 from the old files, but ill look into eliminating the duplication